mlflow/R/mlflow/tests/testthat/test-run.R

```diff
@@ -9,3 +9,40 @@ test_that("mlflow can run and save model", {
   expect_true(dir.exists("mlruns/0"))
   expect_true(file.exists("mlruns/0/meta.yaml"))
 })
+
+
+test_that("mlflow run uses active experiment if not specified", {
+  with_mock(.env = "mlflow", mlflow_get_active_experiment_id = function() {
+    123}, {
+      with_mock(.env = "mlflow", mlflow_cli = function(...){
+        args <- list(...)
+        expect_true("--experiment-id" %in% args)
+        expect_false("--experiment-name" %in% args)
+        id <- which(args == "--experiment-id") + 1
+        expect_true(args[[id]] == 123)
+        list(stderr = "=== Run (ID '48734e7e2e8f44228a11c0c2cbcdc8b0') succeeded ===")
+      }, {
+        mlflow_run("project")
```

Comment:
Suggestion: maybe verify the run ID returned by `mlflow_run` as well? e.g. via `expect_equal(mlflow_run("project"), "48734e7e2e8f44228a11c0c2cbcdc8b0")` here? I guess interestingly R differs from Python here, we may eventually want to break this API & return something akin to Python's [SubmittedRun](https://www.mlflow.org/docs/latest/python_api/mlflow.projects.html#mlflow.projects.SubmittedRun) object from `mlflow_run` in R instead of a run_uuid.

Reply:
The returned run id is hardcoded. I don't think there is a need to test we got it.